client/packages/system/src/Patient/Components/utils.ts

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import { PatientRowFragment, usePatient } from '../api';
+import { useDebounceCallback } from '@common/hooks';
+
+const MIN_CHARS = 3;
+
+export const searchPatient = () => {
+  const patientQueries = usePatient.utils.api();
+  const [patients, setPatients] = useState<PatientRowFragment[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [searchText, setSearchText] = useState('');
+  const [overlimit, setOverlimit] = useState(false);
+
+  useEffect(() => {
+    search('');
+  }, []);
+
+  const search = async (searchValue: string) => {
+    setIsLoading(true);
+    patientQueries.get
+      .list({
+        first: 100,
+        sortBy: { key: 'name', direction: 'asc' },
+        filterBy: {
+          nameAndCode: { like: searchValue },
```

Comment:
Probably want fuzzy search in future for this..

Reply:
yep. #502